crates/ruff_cli/src/commands/format.rs

```diff
@@ -50,13 +50,16 @@ pub(crate) fn format(
     } else {
         FormatMode::Write
     };
+
     let (paths, resolver) = python_files_in_path(&cli.files, &pyproject_config, overrides)?;
 
     if paths.is_empty() {
         warn_user_once!("No Python files found under the given path(s)");
         return Ok(ExitStatus::Success);
     }
 
+    eprintln!("{:?}", &paths);
+
```

Comment:
```suggestion
```

Reply:
Yep, that's gone. I was so confused why my print statements, break points, nothing hit until I realised that it is an entire different entry point.